packages/@lwc/ssr-compiler/src/__tests__/fixtures.spec.ts

```diff
@@ -36,11 +36,9 @@ async function compileFixture({ input, dirname }: { input: string; dirname: stri
             lwcRollupPlugin({
                 targetSSR: true,
                 enableDynamicComponents: true,
-                modules: [
-                    {
-                        dir: modulesDir,
-                    },
-                ],
+                // TODO [#3331]: remove usage of lwc:dynamic in 246
```

Comment:
```suggestion
                // TODO [#3331]: remove usage of lwc:dynamic
```

Clearly we are not good at setting deadlines for ourselves.